packages/framework/src/Component/Setting/SettingValueRepository.php

```diff
@@ -54,12 +54,16 @@ public function copyAllMultidomainSettings($fromDomainId, $toDomainId)
                     WHERE domain_id IS NOT NULL
                         AND domain_id != :commonDomainId
                 )',
-            new ResultSetMapping()
+            [
+                'toDomainId' => $toDomainId,
```

Comment:
I am not really happy about duplicating these keys. But I am woried there is no other way than introduce costants for all of them which may worse readibility... What do you think?

Reply:
I see... introducing a constant is a cleaner solution, but I'm worried about worse readability as you mentioned. 
Now I see the appropriate key in SQL string and in parameter arrays. 

Or I can rewrite those queries to use positional parameters (numerically indexed arrays – no keys), but for me it's more clear as it is right now.